content/rest-api/filter.md

```diff
@@ -160,6 +160,139 @@ To get the products that were updated during the last 4 days, you can use the fo
+| `NOT EMPTY` | no value | Only returns product models that have a family |
+
+#### Examples
+To get the product models that are in the `camcorders` family, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"family":[{"operator":"IN","value":["camcorders"]}]}
+```
+
+To get the product models that are not in the `camcorders` and `digital_cameras` family, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"family":[{"operator":"NOT IN","value":["camcorders","digital_cameras"]}]}
+```
+
+### On creation or update date
+
+To filter product models on creation or update date, use the product property `created` and `updated`, respectively.
+Here are the allowed operators to filter on these properties as well as the corresponding type of value expected in the `search` query parameter.
+
+:::info
+Note that dates are interpreted in the time zone of the server that runs Akeneo Pim (e.g. date.timezone setting in php.ini).
+:::
+
+| Operator | Allowed value type | Filter description |
+| ----------------- | -------------- | ------------------ |
+| `=` | datetime <br> _Format: YYYY-MM-DD hh:mm:ss_ | Only returns product models that were respectively<br> updated or created during the given day |
+| `!=`  | datetime <br> _Format: YYYY-MM-DD hh:mm:ss_ | Only returns product models that were respectively<br> not updated or not created during the given day  |
+| `<` | datetime <br> _Format: YYYY-MM-DD hh:mm:ss_ | Only returns product models that were respectively<br> updated or created before the given day |
+| `>` | datetime <br> _Format: YYYY-MM-DD hh:mm:ss_ | Only returns product models that were respectively<br> updated or created after the given day |
+| `BETWEEN` | array of datetimes <br> _Format: YYYY-MM-DD hh:mm:ss_ | Only returns product models that were respectively<br> updated or created between the two given dates |
+| `NOT BETWEEN` | array of datetimes <br> _Format: YYYY-MM-DD hh:mm:ss_ | Only returns product models that were respectively<br> not updated or not created between the two given dates |
+| `SINCE LAST N DAYS` | integer | Only returns product models that were respectively updated<br> or created during the last n days, n being the given value |
+|
+#### Examples
+To get the product models that were created on the 4th of July 2016 at 10am, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"created":[{"operator":"=","value":"2016-07-04 10:00:00"}]}
+```
+
+To get the product models that were updated during the last 4 days, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"updated":[{"operator":"SINCE LAST N DAYS","value":4}]}
+```
+
+
 ## Filter on product values
 To filter products on its [product values](/documentation/resources.html#product-values), you can use the `search` query parameter when requesting products. The value given to this query parameter should be a valid JSON as shown below.
```

Comment:
"To filter products, and product models since the v2.3,"

Reply:
Fix